.github/workflows/test.yml

```diff
@@ -26,6 +26,12 @@ jobs:
     - name: Run all pre-commit checks on the full repo!
       run: |
         pre-commit run --all-files
+    - name: Build and install pykevlar
+      run: |
+        mamba install bazel
+        ./generate_bazelrc
+        bazel build -c dbg //python:pykevlar_wheel
+        pip install --force-reinstall bazel-bin/python/dist/*.whl
```

Comment:
That sounds really nice. My only complaint is that using bazel for pure python stuff tends to reduce my iteration time because I have to run a `bazel build ` whenever I make a change to the python source. By comparison, I would normally install in editable mode with `pip -e` and then there's no need to run an extra step. Is there a good way to ameliorate this?

Reply:
Notebook or regular python file? For a regular python rule, you could write a py_binary rule and just (i)bazel run it. For a notebook, I can cook you up something.

Reply:
Cool, bazel run works fine for just running. 

Other things that I'd probably be looking for:
* like you say, yeah, notebook autoreload is something I use a lot.
* I use the vscode testing tools a lot. Being able to hit a single keyboard shortcut to run the current test or the failing tests, etc is pretty helpful. 
* I use the vscode debugger a lot, from three places: 1) from a test, I'll run it in a debug session. 2) from a notebook, I'll run a cell in debug mode (there's a little dropdown on the left, super handy) 3) run a straight python script, launch into the debugger with F5. 
I don't want to be annoying, but I'm sure people have worked on this and you probably know where to look. 

(All this is besides the point with the original comment here about the CI. With the CI, of course we can run through bazel.)